gef-ui/server/controllers/application-details/index.js

```diff
@@ -151,20 +163,36 @@ const applicationDetails = async (req, res, next) => {
       maker,
     };
 
-    const partial = stateToPartial(application.status, req.url);
-    let params;
+    if (req.url) {
+      url = req.url.split('/');
```

Comment:
any reason why we don't get these values from req.params? we

Reply:
Url is coming from the req and based on previous development it's easier to pick it up from req, changing to params will involve changing other controllers as well unfortunately.